### ISSUE TITLE
Incorrect handling of multiclass labels when using JSON

### ISSUE THREAD

**Issue by rushter** [VowpalWabbit/vowpal_wabbit#2513]
[Wiki](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/JSON) uses integers for class labels, but they don't work as expected for multiclass problems.

```
echo '{"_label":3, "feature":10}\n{"_label":3}' | vw --oaa 7 --json

Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile =
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
label 1077936128 is not in {1,7} This won't work right.
1.000000 1.000000            1            1.0 1077936128        1        2
label 1077936128 is not in {1,7} This won't work right.
1.000000 1.000000            2            2.0 1077936128        1        1

finished run
number of examples = 2
weighted example sum = 2.000000
weighted label sum = 0.000000
average loss = 1.000000
```

Current workaround is to use string type: 
```
echo '{"_label":"3", "feature":10}' | vw --oaa 7 --json
```

But this won't work with weights:
```
echo '{"_label":{"Label":"2","Weight":0.3}}' | vw --oaa 7 --json

Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile =
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
vw example #0(parse_example_json.h:1505): JSON parser error at 22: Terminate parsing due to Handler error.. Handler: Unsupported label property: 'Label' len: 5. The only string value supported in this context is NaN.State: LabelObject
vw (parse_example_json.h:1505): JSON parser error at 22: Terminate parsing due to Handler error.. Handler: Unsupported label property: 'Label' len: 5. The only string value supported in this context is NaN.State: LabelObject
```

Another workaround that seems to work is to include weights in the label:
```
echo '{"_label":"2 0.3"}' | vw --oaa 7 --json
```
#### Environment

`8.8.1 (git commit: 866a3ff) MacOS`

**Comment by jackgerrits** [comment]
Currently the json format does not support JSON based label description for multiclass as you tried. The current workaround is to provide the text based label format as you did.

I have a proposal out for fixing inconsistencies and issues like this is the JSON format. See here: https://github.com/VowpalWabbit/vowpal_wabbit/issues/2488

**Comment by rushter** [comment]
Thanks for your response. Should I close this issue?

**Comment by jackgerrits** [comment]
Added to docs.
